src/backend/dx11/src/device.rs

```diff
@@ -588,15 +589,36 @@ impl Device {
                 }
             }
             image::ViewKind::D2 => {
```

Comment:
Interestingly, even for a 2D texture, the dx11 backend seems to only generate `2DTextureArray` views, so this code doesn't get used even in the `msaa-level` example.

Reply:
yeah, IIRC there was a good reason for it :)

Reply:
I shall defer to experience :)